tests/statistics_cards_verification_test.py

```
@@ -0,0 +1,131 @@
+            
+            # Click on statistics tab to activate it
+            await page.click('[data-tab="statistics"]')
+            await page.wait_for_timeout(2000)  # Allow tab to load
+            
+            # Wait for statistics to load
+            logger.info("⏳ [STATS-CARDS] Waiting for statistics to load...")
+            
+            # Wait for either cards or loading indicator
+            try:
+                await page.wait_for_selector('.model-statistics-table-container, .data-card, .loading', timeout=15000)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)